MessengerOnFileForLinux/External_GTest.cmake

```diff
@@ -0,0 +1,58 @@
+find_package(Threads REQUIRED)
```

Comment:
to jest ten external co mówiłeś żebyśmy nie musieli mieć lokalnie gtesta/gmocka?

Reply:
tak